agent/rpm/pbench-agent.spec.j2

```diff
@@ -48,7 +48,7 @@ Requires:  perl-Time-HiRes
 
 Requires:  bc, bzip2, hostname, iproute, iputils, net-tools
 Requires:  openssh-clients, openssh-server, procps-ng, psmisc, redis
-Requires:  rpmdevtools, rsync, screen, sos, tar, xz
+Requires:  rpmdevtools, rsync, screen, sos, tar, which, xz
```

Comment:
Adding a dependency, even one as  common as `which`, is disappointing and seems like something that we should resist.

Why can't we use `command -v`?  (We already use it in 30 places in our code.)